container-builder/examples/dockerfiles/SonataFlow.dockerfile

```diff
@@ -30,7 +30,7 @@ RUN /home/kogito/launch/build-app.sh ./resources
 #=============================
 # Runtime Run
 #=============================
-FROM registry.access.redhat.com/ubi8/openjdk-11-runtime:latest
+FROM registry.access.redhat.com/ubi8/openjdk-17:latest
```

Comment:
Here is change of base image type, is this change needed? Before was used runtime-only container.

Reply:
This just an example of a potential dockerfile, so I basically unified the configurations.

Reply:
anyway, I have configuerd openjdk-17-runtime instead